cmd/influxd/backup/backup.go

```diff
@@ -360,7 +360,10 @@ Options:
 func retentionAndShardFromPath(path string) (retention, shard string, err error) {
 	a := strings.Split(path, "/")
 	if len(a) != 3 {
-		return "", "", fmt.Errorf("expected database, retention policy, and shard id in path: %s", path)
+		a = strings.Split(path, "\\")
```

Comment:
I'm unfamiliar with what the contents of the path would be, but would `os.PathSeparator` be a better method and do something like:

```
a := srings.Split(path, os.PathSeparator)
```

If not, then we may want to make a specific windows/unix file for this specific function, as it feels the issue is platform specific.